.github/workflows/build-release.yml

```diff
@@ -0,0 +1,86 @@
+# Build docker image, build cumulocity archive and release it to the repository
+
+name: Build & release oee-simulators
+
+on:
+  workflow_dispatch:
+    inputs:
+        release_body:
+          type: string
+          description: GitHub Release Message
+          default: "GitHub Release Message"
+  push: 
+    branches: [ COE-722 ]
+
+env:
+  BUILD_OFFSET: 25
```

Comment:
maybe we run the workflow 15 more time, then the build number will be higher than 30 (and therefore higher than the last release) and we can get rid of this ;-)

Reply:
Seems reasonable 🦖